Helloworld.java

```diff
@@ -24,7 +24,7 @@ public static void main(String[] args) {
 		System.out.println("Here's looking at you, kid.");
 		System.out.println("Welcome to New York.");
 		System.out.println("That's one small step for a man, one giant leap for mankind.");
-		System.out.println("I think; therefore, I am confused.");
+		System.out.println("When you come to a fork in the road, take it.");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)